virttest/qemu_devices/qcontainer.py

```diff
@@ -871,11 +871,9 @@ def get_aavmf_vars(params):
             devices.append(qdevices.QStringDevice('AAVMF_CODE',
                                                   cmdline=aavmf_code))
             aavmf_vars = get_aavmf_vars(params)
-            if not os.path.exists(aavmf_vars):
-                logging.warn("AAVMF variables file '%s' doesn't exist, "
-                             "recreating it from the template (this should "
-                             "only happen when you install the machine as "
-                             "there is no default boot in EFI!)", aavmf_vars)
+            force_create = params.get("force_create_image_image1",
+                                      params.get("force_create_image"))
```

Comment:
@ldoktor  could you explain why you use the value of "force_create_image" as the default value for "force_create_image_image1" ?

Reply:
Basically the `force_create_image` is the parameter, I'm interested in. This parameter can be globally enabled, but locally disabled per-object-name. Theoretically I should use `params.object_params("image1")` which would create a copy of params replacing values of keys with keys ending with `_image1`. As I'm only interested in this single parameter I think it makes sense to simply get the `_image1` value and if not available fallback to the overall value.

Reply:
ok, got you, thx!
I am puzzled if we need to use fresh AAVMF_VARS.fd for other image, not only image1.
Get the value of "force_create" according to the `force_create_image_$image_name`
When run with a test loop, include many cases, and these case use may use same image name, such as `sn1` in snapshot related case. and the image was force created in the new case,   In this situation, `aavmf_vars` already exist.